### PR TITLE
replace hashibot with github actions

### PR DESCRIPTION
**Pull request by megan07**
Fixes https://github.com/hashicorp/terraform-provider-google/issues/9136

### CODE REVIEW

File: .github/workflows/issue-comment-created.yml
Comment (bflad) [inline .github/workflows/issue-comment-created.yml:11]: This was my bad in the migration plan, I don't think this is necessary for this workflow 👍 

```suggestion
```

File: .github/workflows/pull-request-size.yml
Comment (bflad) [inline .github/workflows/pull-request-size.yml:10]: Upstream enhancement to allow specifying the size labels is still pending: https://github.com/CodelyTV/pr-size-labeler/pull/26

Up to you folks whether this is worth skipping this migration for now (I will ensure there is a solution before decommissioning) or if it is worth renaming the repository `size/*` labels with lowercase to match this action after merging this.

File: .github/workflows/issue-comment-created.yml
Comment (rileykarson) [inline .github/workflows/issue-comment-created.yml:1]: This will just remove `stale` and `waiting-reply` if a maintainer adds them at creation time, right? Is it possible a maintainer would do that on first file intentionally / do we expect it to happen unintentionally?

**Review by bflad — APPROVED** [review]

File: .github/workflows/issue-comment-created.yml
Comment (megan07) [reply .github/workflows/issue-comment-created.yml:1]: i think it's when a comment is created on the issue that we'll remove `stale` and `waiting-reply`, does that answer your question?

Comment (rileykarson) [reply .github/workflows/issue-comment-created.yml:1]: Oh, yup, that makes more sense!

**Review by rileykarson — APPROVED** [review]

File: .github/workflows/pull-request-size.yml
Comment (megan07) [reply .github/workflows/pull-request-size.yml:10]: we have no preference, so i'll rename the labels to lowercase. thanks!

**Pull request merged**
